browser/mapping.go

```diff
@@ -211,7 +215,10 @@ func mapElementHandle(vu k6modules.VU, eh api.ElementHandle) mapping {
 //
 //nolint:funlen
 func mapFrame(vu k6modules.VU, f api.Frame) mapping {
```

Comment:
In an attempt to make xk6-browser follow go good practices, could we not pass in `ctx` as the first param here (and all other mappings methods)?

```
func mapFrame(ctx, context.Context, vu k6modules.VU, f api.Frame) mapping {
```

That way we also don't need to wrap the `k6modules.VU` in a `moduleVU`. Please let me know if i've missed something here.

Reply:
I've done it like so as it's kind of a k6 standard, and it's less code :) Passing ctx as the first parameter is a dogma in the Go world. You can even see core Go developers use the embedded context pattern everywhere in their code. Then again, I'll think about this more and get back to you. Thanks for your review! 🙇

Reply:
I agree that passing around `ctx` is tedious, but it's less confusing than when we end up with multiple `ctx`s. We should only ever need to work with one `ctx`. 🙂